scripts/travis/build.sh

```diff
@@ -76,7 +76,7 @@ cd $build_path
 #
 progress "Configuring with cmake"
 
-cmake_flags="-DARB_WITH_ASSERTIONS=ON -DARB_WITH_MPI=${WITH_MPI} -DARB_WITH_PYTHON=${ARB_WITH_PYTHON} ${CXX_FLAGS} ${PY_FLAGS}"
+cmake_flags="-DARB_WITH_ASSERTIONS=ON -DARB_WITH_MPI=${WITH_MPI} -DARB_WITH_PYTHON=${ARB_WITH_PYTHON} -DARB_ARCH=$1 ${CXX_FLAGS} ${PY_FLAGS}"
```

Comment:
I think that the `ARCH` variable should be propogated into this script, so you could say `-DARB_ARCH=${ARCH}`

Reply:
Oh I didn't know that. Thanks!